content/package.json

```diff
@@ -9,8 +9,8 @@
     "start:db": "node ./dist/src/entrypoints/run-local-database.js",
     "copy-test-resources": "yarn cpy test/integration/resources/* dist --parents",
     "test": "run-p 'test:*'",
-    "test:unit": "yarn build && jasmine --config=jasmine.unit.json",
-    "test:integration": "yarn copy-test-resources && jasmine --config=jasmine.integration.json"
+    "test:unit": "yarn build && jasmine-ts --config=jasmine.unit.json",
```

Comment:
Do we still need to run the build here? If that's the case, then why is not needed during integration?